src/server/config/schema.js

```diff
@@ -197,6 +197,11 @@ export default () => Joi.object({
       }))
     }))
   }).default(),
+
+  savedObjects: Joi.object({
+    indexCheckTimeout: Joi.number().default(5000)
```

Comment:
The issue description mentioned the wait defaulting to a second. Just want to ensure it wasn't an oversight.

Reply:
I waffled around a bit on the timeout, ended up with 5 seconds though, forgot to update the description